sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1339,20 +1396,38 @@ def task_worker(self):
                            api.tx_disable_channel(host_lanes, True)
                            self.port_dict[lport]['cmis_state'] = self.CMIS_STATE_READY
                            continue
+                    # Configure the target output power if ZR module
+                        if api.is_coherent_module():
+                           tx_power = self.port_dict[lport]['tx_power']
+                           # Prevent configuring same tx power multiple times
+                           if 0 != tx_power and tx_power != api.get_tx_config_power():
```

Comment:
what does it mean when tx_power == 0?

Reply:
it means no 'tx_power' missing in the PORT table of CONFIG_DB. See get_configured_tx_power()